frontend/src/components/groups/Group.tsx

```diff
@@ -149,16 +143,25 @@ export function Group() {
 								setAddMemberModalOpen(true);
 							}}
 							endIcon={<PersonAddAlt1Icon />}
-							sx={{ marginRight: "0.5em", width: "auto" }}
 						>
 							Add Member
 						</Button>
 					</AuthWrapper>
-					<AuthWrapper currRole={role} allowedRoles={["owner", "editor"]}>
-						<EditMenu setDeleteGroupConfirmOpen={setDeleteGroupConfirmOpen} />
```

Comment:
Any reason you are get rid of the edit menu which includes "edit group names etc"?

Reply:
I think this was a mistake somehow. I will fix this and check to see what else is missing from main.